TradeItIosTicketSDK2/TradeItLinkBrokerUIFlow.swift

```diff
@@ -46,10 +49,12 @@ class TradeItLinkBrokerUIFlow: NSObject, TradeItWelcomeViewControllerDelegate, L
         onNavigationController navController: UINavigationController,
         asRootViewController: Bool,
         showWelcomeScreen: Bool,
+        hideOpenAccountButton: Bool,
```

Comment:
Yeah that's a good reason but I think in this case the person calling the function is more likely to get confused with the `show` `hide` interface than us in the code below